realm/realm-library/build.gradle

```diff
@@ -46,6 +46,7 @@ repositories {
 
 dependencies {
     provided 'io.reactivex:rxjava:1.1.0'
+    compile "com.android.support:recyclerview-v7:23.1.1"
```

Comment:
With provided keyword I have got strange gradle error that says there are duplicated tasks related to recycler-view support library. It might be issue  [Support provided AARs](https://code.google.com/p/android/issues/detail?id=149268). I think now it is not possible to use provider with .aar packages.